src/message/socket-wrapper.js

```diff
@@ -40,6 +42,53 @@ var SocketWrapper = function( socket, options ) {
 };
 
 utils.inherits( SocketWrapper, EventEmitter );
+SocketWrapper.lastPreparedMessage = null;
```

Comment:
I mean, it's kind of hard to describe it anymore than what the code and the name does. It's kind of obvious.